src/pal/src/eventprovider/lttng/eventprovdotnetruntimerundown.cpp

```diff
@@ -0,0 +1,1640 @@
+        Error = ERROR_SUCCESS;
+                    }
+    }
+
+
+return Error;
+}
+extern "C" BOOL  EventEnabledModuleDCStart(){ return TRUE;}
+extern "C" ULONG  FireEtwModuleDCStart(
+const unsigned __int64  ModuleID,
+const unsigned __int64  AssemblyID,
+const unsigned int  ModuleFlags,
+const unsigned int  Reserved1,
+PCWSTR  ModuleILPath,
+PCWSTR  ModuleNativePath
+)
+{
+ULONG Error = ERROR_WRITE_FAULT;
+    INT ModuleILPath_path_size = 0;
+    CHAR ModuleILPath_full_name[ MAX_PATH + 1 ];
+    ModuleILPath_path_size = WideCharToMultiByte( CP_ACP, 0, ModuleILPath, -1, ModuleILPath_full_name,
+                                             MAX_PATH, NULL, NULL );
+    INT ModuleNativePath_path_size = 0;
+    CHAR ModuleNativePath_full_name[ MAX_PATH + 1 ];
+    ModuleNativePath_path_size = WideCharToMultiByte( CP_ACP, 0, ModuleNativePath, -1, ModuleNativePath_full_name,
+                                             MAX_PATH, NULL, NULL );
+                if( ModuleILPath_path_size != 0 ){
+            if( ModuleNativePath_path_size != 0 ){
+
+                                
+     tracepoint(
+        dotnetruntimerundown,
+        ModuleDCStart,
+        ModuleID,
+        AssemblyID,
+        ModuleFlags,
+        Reserved1,
+        ModuleILPath_full_name,
+        ModuleNativePath_full_name
+        );
+
+        Error = ERROR_SUCCESS;
+                    }
+    }
+
+
+return Error;
+}
+extern "C" BOOL  EventEnabledModuleDCStart_V1(){ return TRUE;}
+extern "C" ULONG  FireEtwModuleDCStart_V1(
+const unsigned __int64  ModuleID,
+const unsigned __int64  AssemblyID,
+const unsigned int  ModuleFlags,
+const unsigned int  Reserved1,
+PCWSTR  ModuleILPath,
+PCWSTR  ModuleNativePath,
+const unsigned short  ClrInstanceID
+)
+{
+ULONG Error = ERROR_WRITE_FAULT;
+    INT ModuleILPath_path_size = 0;
+    CHAR ModuleILPath_full_name[ MAX_PATH + 1 ];
+    ModuleILPath_path_size = WideCharToMultiByte( CP_ACP, 0, ModuleILPath, -1, ModuleILPath_full_name,
+                                             MAX_PATH, NULL, NULL );
+    INT ModuleNativePath_path_size = 0;
+    CHAR ModuleNativePath_full_name[ MAX_PATH + 1 ];
+    ModuleNativePath_path_size = WideCharToMultiByte( CP_ACP, 0, ModuleNativePath, -1, ModuleNativePath_full_name,
+                                             MAX_PATH, NULL, NULL );
+                if( ModuleILPath_path_size != 0 ){
+            if( ModuleNativePath_path_size != 0 ){
+
+                                
+     tracepoint(
+        dotnetruntimerundown,
+        ModuleDCStart_V1,
+        ModuleID,
+        AssemblyID,
+        ModuleFlags,
+        Reserved1,
+        ModuleILPath_full_name,
+        ModuleNativePath_full_name,
+        ClrInstanceID
+        );
+
+        Error = ERROR_SUCCESS;
+                    }
+    }
+
+
+return Error;
+}
+extern "C" BOOL  EventEnabledModuleDCStart_V2(){ return TRUE;}
+extern "C" ULONG  FireEtwModuleDCStart_V2(
+const unsigned __int64  ModuleID,
+const unsigned __int64  AssemblyID,
```

Comment:
Same stylistic comment here.  This will be much easier to read (and debug) with proper formatting.